scripts/reinstall.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+DIVIDER="----------------------------------------------------------------------"
+clear
+echo "$DIVIDER"; echo "-- ATAT Web API developer reinstall"; echo "$DIVIDER"
+echo "This script is for developer convenience. It assumes:"
+echo " - \$ATAT_WEB_API_DIR ($ATAT_WEB_API_DIR) is defined"
+echo " - nvm is installed and \$NVM_DIR ($NVM_DIR) is defined"
+echo "This will do the following:"
+echo " - operate from the currently selected git branch and state"
+echo " - install the preferred node version specified in .nvmrc"
+echo " - install the latest AWS CLI"
+echo " - install the latest AWS CDK Toolkit"
+echo " - install the latest TypeScript"
+echo " - install all package dependencies"
+echo " - bootstrap, build, and test"
+echo " - display software tool versions"
+read -rp "Press enter to continue or Ctrl+C to cancel"
+. $NVM_DIR/nvm.sh
+. $HOME/.profile
+. $HOME/.bashrc
+echo; echo "[Install AWS CLI]" && sudo apt install awscli
```

Comment:
As @juliusfitzhugh-ccpo pointed out, this will break on macOS; though, not just there but any Linux that either doesn't use apt or that doesn't package the AWS CLI. We also want to encourage folks to use AWS CLI v2 and some distros may still be packaging v1.

Reply:
Good points.  I've removed the `awscli` installation from the replacement `install.sh`